src/index.html

```diff
@@ -11,7 +11,170 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post">
+      <div class="fieldset_style">
+        <fieldset>
+          <legend>Personal info</legend>
+
+          <div>
+            <label>
+              Surname:
+              <input
+              type="text"
+              name="Surname"
+              autocomplete="off"
+              required
+              >
+            </label>
+          </div>
+
+          <div class="field_form">
+            <label>
```

Comment:
Just add class to label and use proper styles there, no need to create extra wrapper here